packages/nuekit/src/util.js

```diff
@@ -1,6 +1,7 @@
 /* misc stuff. think shame.css */
 
 import { execSync } from 'node:child_process'
+import { realpathSync } from 'node:fs'
 import { promises as fs } from 'node:fs'
```

Comment:
```suggestion
import { promises as fs, realpathSync } from 'node:fs'
```
How about merging import statements?